run/testing/jobs.e2e_test.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cloudruntests
+
+import (
+	"testing"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/cloudrunci"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestCloudRunJobs(t *testing.T) {
+	tc := testutil.EndToEndTest(t)
+
+	crj := cloudrunci.NewJob("runjobs", tc.ProjectID)
+	crj.Dir = "../jobs"
+	crj.AsBuildpack = true
+	crj.Env = cloudrunci.EnvVars{
+		"FAIL_RATE": "0.0",
+		"SLEEP_MS":  "10000",
+	}
+
+	if err := crj.Create(); err != nil {
+		t.Fatalf("Create %q: %v", crj.Name, err)
+	}
+	if err := crj.Run(); err != nil {
+		t.Errorf("Run(%s): %s", crj.Name, err)
+	}
+	defer crj.Clean()
+
```

Comment:
I would love to see a check for the logs that the job creates (a log should include "Task").

Reply:
Maybe the `Completed Task` check was added since this comment? Note that under the hood, LogEntries is already filtering to logs associated with this Job as a resource.

Reply:
Yes, this check was added in response, and i forgot to close out the conversation.